core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/bundle/BundleRestService.java

```diff
@@ -78,59 +80,61 @@ public Promise<Bundles> searchByGet(@ParameterObject final BundleRestSearch para
 	}
 	
 	@Operation(
-		summary="Retrieve bundles", 
-		description="Returns a collection resource containing all/filtered registered bundles."
-			+ "<p>Results are by default sorted by ID."
-			+ "<p>The following properties can be expanded:"
-			+ "<p>"
-			+ "&bull; resources() &ndash; this list of resources this bundle contains"
+		summary="Retrieve a list of bundles", 
+		description="""
+			Returns a collection resource containing all/filtered registered bundles. Results are sorted by ID by default.
+			
+			The following properties can be expanded:
+			
+			* `resources()` - the list of resources this bundle directly contains
+		"""
 	)
 	@ApiResponses({
 		@ApiResponse(responseCode = "200", description = "OK"),
-		@ApiResponse(responseCode = "400", description = "Invalid search config"),
+		@ApiResponse(responseCode = "400", description = "Bad Request"),
 	})
 	@PostMapping(value="/search", produces = { AbstractRestService.JSON_MEDIA_TYPE })
 	public Promise<Bundles> searchByPost(@RequestBody(required = false) final BundleRestSearch params) {
 		return searchByGet(params);
 	}
 	
 	@Operation(
-		summary="Retrieve bundle by its unique identifier",
-		description="Returns generic information about a single bundle associated to the given unique identifier."
+		summary="Retrieve bundle by idr",
```

Comment:
Is this an extra letter or did a longer sentence get truncated?

Reply:
Typo! Thanks!